proknow/Patients/Entities/EntitySummary.py

```diff
@@ -68,17 +70,56 @@ def get(self):
                 entities = [entity.get() for entity in patient.find_entities(lambda entity: True)]
         """
         entity_type = self._data["type"]
+        DELAY = 0.2
+        MAX_COUNT = 5 / DELAY
         if entity_type == "image_set":
-            _, image_set = self._requestor.get('/workspaces/' + self._workspace_id + '/imagesets/' + self._id)
+            count = 0
```

Comment:
Shouldn't `count` be set to 0 outside of the `if` so all entity types have the value initialized?

Reply:
Yes, that was a mistake. It should have been set with the other variables at the top. Fixed.